### DISCUSSION TITLE
SSH open on Slingbox

### DISCUSSION THREAD

**Discussion by skijohn877** [GerryDazoo/Slinger#551] [Q&A]
So I changed out my router the other day. Knew I was going to get a new ip address/port for my Slingbox 350. I ran a port map against the new ip which showed a new port 5214 was open. No issues using new ip/port in the config.ini file. However I did notice that port 22 SSH was open as well. I was able to SSH to the the slingbox but no username/password combo worked.

Was anyone able to successfully ssh to a slingbox?

**Comment by pfuller88** [comment]
I noticed this several months back and ran several brute force programs, trying to access via port 22.   No luck.
